.github/workflows/stale-bot.yml

```
@@ -0,0 +1,21 @@
+name: 'Label stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * 1' # runs once a week
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: '' # no comment left if string is empty
+          stale-pr-message: '' # no comment left if string is empty
+          days-before-stale: 30
+          days-before-close: -1
+          stale-issue-label: 'needs attention'
+          stale-pr-label: 'needs attention'
+          exempt-issue-labels: 'good intro to dask,good first issue,Good First Issue,good second issue,feature request'
```

```suggestion
          exempt-issue-labels: 'good first issue,good second issue,feature,flaky test, enhancement'
```

Some of the labels in here do not exist

---

yeah I imagine they either used to exist or they exist on dask but not distributed.